### ISSUE TITLE
Multi-user System

### ISSUE THREAD

**Issue by gangesh** [maxpozdeev/mytinytodo#51]
Any plans to update script for multi-user?

Suggestion
1. Role based system. e.g. certain roles have full access or set limited based on group of lists. 
2. Roles can have capabilities, Full access, Edit only for existing lists, Read only etc
3. Comment system  for each task.

**Comment by wisesiteguy** [comment]
Definitely a ++++ from me to see this functionality used. It would be great to have an entire team able to use the identical system

**Comment by maxpozdeev** [comment]
Little info about roadmap.
v1.8 - add changes in database table to remove useless fields.
v1.9 - add support of postgresql (or maybe in v2.0)
v2.0 - add support of multiple users, but without any collaboration things.
...

**Comment by wisesiteguy** [comment]
Excited for V2.0 Max. Shout if you need an alpha tester!

**Comment by grahn195** [comment]
Vote up for multiple user !

**Comment by chupchupchup** [comment]
yes ! or use the Login Password from wordpress ! (using same table)

**Comment by Matt6464** [comment]
Hi Max, a lot of months I am watching your project, it seems a great tool, and with multi users it would be exactly what I need ! :)
Do you think you will release this enhancement quickly ?
